server.py

```diff
@@ -112,7 +124,7 @@ def auth_token():
             return jsonify({'error': 'Unsupported authorization scheme.'})    
 
     else: # route not in open/close routes
-        logging.debug("Route doesn't exist.")
+        logging.debug(f"Route not found: {request_key}")
```

Comment:
I'm a new engineer, I've implemented a new API and just spent 2 hours trying to understand wtf I get 404 when I try to invoke it :)